pkg/app/app.go

```diff
@@ -40,7 +41,7 @@ func NewCR(c Config) *applicationv1alpha1.App {
 			Namespace:   "giantswarm",
 			Annotations: annotations,
 			Labels: map[string]string{
-				"app-operator.giantswarm.io/version": "1.0.0",
+				"app-operator.giantswarm.io/version": c.VersionLabelValue,
```

Comment:
Is there any use case for not setting it to 0.0.0?

Reply:
I checked and this library is only used by opsctl. So we don't need to make this configurable.

Reply:
It’s also (even more importantly) used by architect. But we need 0.0.0 there as well

Reply:
Thanks I think we sometimes use opsctl in CI which confused me. But makes sense architect needs this too. https://github.com/giantswarm/architect/pull/446